### PR TITLE
added event description. remove user and add user not working properly

### PR DESCRIPTION
**Pull request by cliffplaysdrums**
@ayo test this code as soon as you can please. I added an event description (need to fix layout a little too), but I noticed that something is going on with keeping track of users. When i create a new user, I see an error in the console output and no confirmation in the application. If i hit create user again it says, "Username is taken," but if I try to log in with that user, it opens the application twice. If i delete a user, it stills says username taken. If you can verify that it's not the changes I made causing that, then go ahead and merge. Either way we have to fix it.

### CODE REVIEW

**Comment by afafore1** [comment]
I tried to create a new user now and I did not have any problems..

**Comment by cliffplaysdrums** [comment]
@afafore1 Try to remove that user and create another with the same name.

**Comment by afafore1** [comment]
Yeah I'll try that now

**Comment by afafore1** [comment]
I am able to add the user again, however I get an error because it doesn't remove the user from the database. I'll fix that now

**Comment by cliffplaysdrums** [comment]
Okay, that's what I was wondering. Hopefully I added the event description correctly every where.
I've been running into problems trying to create a notification, but I'll try again tomorrow.

**Comment by afafore1** [comment]
Okay. I'll also look into that.

**Comment by cliffplaysdrums** [comment]
Okay cool

**Comment by afafore1** [comment]
Your code looks fine to me. I'll test it and see

File: src/scheduler/GUI.java [hunk 1]
```diff
@@ -946,7 +946,7 @@ private void buildDateGUI() {
                 c.gridy = (hour + 1) * 2;
```
Comment (afafore1) [inline src/scheduler/GUI.java:946]: Interesting... 
For deleting event... when user clicks on date, they see descriptions but can't click yes ?
Should "Delete" be changed to "Delete All" ?
We can then make something where you can search and delete event

Comment (afafore1) [inline src/scheduler/GUI.java:946]: I'll take care of delete function

**Comment by cliffplaysdrums** [comment]
@afafore1 okay. we can make it a search instead if that's easier. with the little time we have left, i'd just like to have the functionality regardless of where it is or how it's implemented.
Are you okay with merging this branch?

**Pull request merged**